src/components/ConnectedList.jsx

```diff
@@ -7,6 +7,7 @@ import { translate } from 'cozy-ui/react/I18n'
 import KonnectorList from './KonnectorList'
 
 import addAccount from '../assets/icons/icon-plus.svg'
+import noKonnectors from '../assets/images/connected-accounts.svg'
```

Comment:
At first read I have bee thinking that `noKonnectors` was a boolean. What do you think about using something like `noKonnectorsPicture` or `pictureForEmtpyList` ?

I missed the fist time but maybe we should to the same for `addAcccount` right above and use something like `addAccountIcon`.